src/ol_infrastructure/infrastructure/mongodb_atlas/Pulumi.infrastructure.mongodb_atlas.mitx.QA.yaml

```diff
@@ -0,0 +1,15 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-qa
+encryptedkey: AQICAHijXuVxVlAL6bY9xCOrzO3YYhFlQBPt6jNyJGkhYu+q4QFqnCZkB/FKA+NYkHX2ontQAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMv9SWZurZCB8X84eFAgEQgDuJIDAG4l2pwKedRX7FidwguT271f1PEXxrVcWJ8BxdpQxLaO/zUSxd+mRR+DHbg6FZZNp+pnM5c1x7WQ==
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-mitx-qa.odl.mit.edu
+  consul:scheme: https
+  environment:business_unit: residential
+  environment:target_vpc: residential_mitx_vpc
+  mongodb_atlas:disk_autoscale_max_gb: '200'
+  mongodb_atlas:disk_size_gb: '100'
```

Comment:
Is this gonna be enough?

Reply:
That should be enough to start with. It will scale up to 200 GB if needed.